docs/dev/framework/image-processing/_index.md

```diff
@@ -21,7 +22,8 @@ output images and metadata for your templates.
 |-|-|-|
 | Output images in a template | [&rarr;&nbsp;Image Studio][ImageStudio] | high |
 | Resize images with full control over each parameter | [&rarr;&nbsp;ImageFactory][ImageFactory]<br>[&rarr;&nbsp;PictureFactory][PictureFactory] | medium | 
-| Use the image processing features outside a Contao application | [&rarr;&nbsp;contao/image][LibraryImage]<br>[&rarr;&nbsp;contao/imagine&#8209;svg][LibraryImagine] | low |
+| Use the image processing features outside a Contao application | [&rarr;&nbsp;contao/image][LibraryImage] | medium rare |
```

Comment:
“medium rare” wasn’t really meant serious ☺️
Should it be just “medium” or something other between medium and low?

Reply:
I still had a good laugh. And now it's forever in the history. :smile: 

Seriously: I changed it to "low to medium" in 45fb7067cb7abe72b5fa7660dc2cb1985b55a25f.